tests/models.ts

```diff
@@ -63,3 +63,5 @@ export type ViewFacetMetadata =
 export type BookPageViewMode = '1up' | '2up' | 'thumb';
 
 export type ChannelSelector = 'Player' | 'Webamp';
+
+export type UserType = 'privs' | 'patron';
```

Comment:
QUESTION: Does there need to be a UserType value for when the user is not logged in?